style.css

```diff
@@ -325,4 +324,120 @@
     padding-top: 1rem;
     font-weight: bold;
   }
+
+  #form {
+    background: #6070ff;
+    width: 400px;
+    padding: 1em;
+    border: 1px solid #fff;
+    border-top-left-radius: 5rem;
+    background-image: url(images/contact\ form\ shapes-mobile2.svg);
+    background-repeat: no-repeat;
+    background-position: right -1px top 10px;
```

Comment:
- [ ] Please layout the elements in the form section using flexbox

Reply:
added